File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/CommentsById.java
package com.example.vkwall.data.model.GetWallByID;

import com.google.gson.annotations.SerializedName;

public class CommentsById {
    @SerializedName("count")
    private Integer count;
    @SerializedName("can_post")
    private Integer canPost;
    @SerializedName("groups_can_post")
    private Boolean groupsCanPost;
    @SerializedName("can_close")
    private Integer canClose;

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public Integer getCanPost() {
        return canPost;
    }

    public void setCanPost(Integer canPost) {
        this.canPost = canPost;
    }

    public Boolean getGroupsCanPost() {
        return groupsCanPost;
    }

    public void setGroupsCanPost(Boolean groupsCanPost) {
        this.groupsCanPost = groupsCanPost;
    }

    public Integer getCanClose() {
        return canClose;
    }

    public void setCanClose(Integer canClose) {
        this.canClose = canClose;
    }

}

File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/CommentsById.kt
package com.example.vkwall.data.model.GetWallByID

import com.google.gson.annotations.SerializedName

class CommentsById {
    @SerializedName("count")
    var count: Int? = null
    @SerializedName("can_post")
    var canPost: Int? = null
    @SerializedName("groups_can_post")
    var groupsCanPost: Boolean? = null
    @SerializedName("can_close")
    var canClose: Int? = null

}
